### PR TITLE
Upgrade Statie 3 → 5.3

### PR DESCRIPTION
**Pull request by TomasVotruba**
I've noticed you might use an upgrade :)

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Statie now supports Twig, let me know if you preffer it.

I personally do, because it's easier to understand in world wide PHP community.

File: site/_posts/2018/2018-11-07-whats-new-in-php-ml-0-7-0.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+id: 2
 title: What's new in PHP-ML 0.7.0
 description: "A quick overview of new features, changes and fixes in PHP-ML 0.7.0 version"
 sources:
-	-
-		- "https://github.com/php-ai/php-ml/blob/master/CHANGELOG.md#070---2018-11-07"
-		- PHP-ML Changelog
+    - 
+        - "https://github.com/php-ai/php-ml/blob/master/CHANGELOG.md#070---2018-11-07"
+        - PHP-ML Changelog
```
Comment (TomasVotruba) [inline site/_posts/2018/2018-11-07-whats-new-in-php-ml-0-7-0.md]: Yaml syntax is used now

File: site/_posts/2018/2018-11-07-whats-new-in-php-ml-0-7-0.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-layout: post
+id: 2
```
Comment (TomasVotruba) [inline site/_posts/2018/2018-11-07-whats-new-in-php-ml-0-7-0.md]: Every post needs own `id` to better work with inter-connections, like `relatedItems` filter etc.

File: site/_layouts/post.latte [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{extends "default"}
+{extends "_layouts/default.latte"}
```
Comment (TomasVotruba) [inline site/_layouts/post.latte]: Full relative paths are used now, to prevent "default" vs "default" file conflicts

File: ecs.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+imports:
+    - { resource: "vendor/symplify/easy-coding-standard/config/common.yml" }
+    - { resource: "vendor/symplify/easy-coding-standard/config/php70.yml" }
+    - { resource: "vendor/symplify/easy-coding-standard/config/php71.yml" }
+    - { resource: "vendor/symplify/easy-coding-standard/config/psr2.yml" }
```
Comment (TomasVotruba) [inline ecs.yml]: ECS uses yaml for many versions

File: composer.json [hunk 5]
```diff
@@ -13,21 +13,19 @@
...
     },
     "scripts": {
       "check": [
         "@ecs",
         "@phpstan"
       ],
-      "ecs": "vendor/bin/ecs check src --config coding-standard.neon",
-      "fix": "vendor/bin/ecs check --fix src --config coding-standard.neon",
+      "ecs": "vendor/bin/ecs check src",
+      "fix": "vendor/bin/ecs check --fix src",
```
Comment (TomasVotruba) [inline composer.json]: This config is loaded by default

File: composer.json [hunk 6]
```diff
@@ -13,21 +13,19 @@
...
-        "nette/utils": "^2.4",
-        "nette/finder": "^2.4",
-        "symplify/statie": "^3.0"
+        "symplify/statie": "^5.3"
     },
     "require-dev": {
-        "phpstan/phpstan": "^0.9.1",
-        "symplify/easy-coding-standard": "^3.0"
+        "phpstan/phpstan": "^0.10",
+        "symplify/easy-coding-standard": "^5.3"
```
Comment (TomasVotruba) [inline composer.json]: While I'm at it :)

File: statie.yml [hunk 7]
```diff
@@ -19,12 +19,7 @@ parameters:
...
+    # see https://www.statie.org/docs/generators/
     generators:
         posts:
-          variable: 'post'
-          variable_global: 'posts'
-          path: '_posts'
-          layout: 'post'
-          route_prefix: ''
-          object: Symplify\Statie\Renderable\File\PostFile
+            route_prefix: ''
```
Comment (TomasVotruba) [inline statie.yml]: Other values are default, no need to fill them

**Comment by akondas** [comment]
> Statie now supports Twig, let me know if you preffer it.
> 
> I personally do, because it's easier to understand in world wide PHP community.

Yup, Twig :+1:

**Comment by akondas** [comment]
Thanks Tomas :+1: :heart_eyes:

**Pull request merged**